src/main/java/virtual_pet/VirtualPet.java

```diff
@@ -0,0 +1,46 @@
+package virtual_pet;
+public class VirtualPet {
+
+    public String petName;
+    public int hunger;
+    public int thirst;
+    public int energy;
+
+    public VirtualPet(String petName, int hunger, int thirst, int energy) {
+
+        this.petName = petName;
+        this.hunger = hunger;
+        this.thirst = thirst;
+        this.energy = energy;
+    }
+    public void changeHunger(int h){
+        hunger += h;}
+    public void changeThirst(int t){
+        thirst += t;
+    }
+    public void changeEnergy(int e){
```

Comment:
When you pass the value of hunger into the constructor and make that variable private, it belongs to the class. Therefore, you do not have to provide a parameter to changeEnergy(). Your method function would like like the following:

public void changeEnergy(){
this.energy += 2;
}